tests/features/news.feature

```diff
@@ -4,9 +4,12 @@ Feature: News content creation
   As an editor
   I need to be able to create and see news items
 
-  @cleanup:media
+  @javascript @cleanup:media
```

Comment:
`@javascript` should be removed

Reply:
The Contacts field uses javascript, we need the tag to use it on the test.